build.xml

```diff
@@ -794,7 +794,7 @@
             description="Generate Thrift Java artifacts">
       <echo>Generating Thrift Java code from ${basedir}/interface/cassandra.thrift...</echo>
       <exec executable="thrift" dir="${basedir}/interface" failonerror="true">
-        <arg line="--gen java:hashcode" />
+        <arg line="--gen java" />
```

Comment:
This argument was removed in `0.9.2` for [THRIFT-2263](https://issues.apache.org/jira/browse/THRIFT-2263) via https://github.com/apache/thrift/commit/acdac816659c88e7b8b601b4ad42dc43bf7d48e2

Before `0.9.2` thrift's java impl did not generate `equals()` and `hashcode()` methods for classes by default unless the `hashcode` flag was set. As of `0.9.2` those methods are generated by default.